src/lib/uniforms-surfnet/src/TimestampField.tsx

```diff
@@ -0,0 +1,77 @@
+import { EuiDatePicker, EuiFormRow, EuiText } from "@elastic/eui";
+import { FieldProps } from "lib/uniforms-surfnet/src/types";
+import moment from "moment";
+/*
+ * Copyright 2019-2022 SURF.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import React from "react";
+import { connectField, filterDOMProps } from "uniforms";
+
+export type TimestampFieldProps = FieldProps<
+    number,
+    { max?: number; min?: number; showTimeSelect: boolean; locale?: string; dateFormat?: string; timeFormat?: string }
+>;
+
+function TimestampField({
+    disabled,
+    id,
+    inputRef,
+    label,
+    description,
+    max,
+    min,
+    showTimeSelect,
+    locale,
+    dateFormat,
+    timeFormat,
+    name,
+    onChange,
+    readOnly,
+    placeholder,
+    value,
+    error,
+    showInlineError,
+    errorMessage,
+    ...props
+}: TimestampFieldProps) {
```

Comment:
That's  a lot of params :)

Reply:
Yes: it was done to keep locale shizzle in the backend. Now the frontend uses "nl-nl" in a couple of places that handle dates. (Process table, Subscription table)

Backend contains some tests also:
https://github.com/workfloworchestrator/pydantic-forms/commit/f78ad6afe5a0b0c72fa08fb66e5ea3dde4ebc88e

It might be good to create a ticket to remove the remaining hardcoded "nl-nl" stuff from the frontend?

Reply:
> It might be good to create a ticket to remove the remaining hardcoded "nl-nl" stuff from the frontend?

Yeah agree. Maybe as an issue here on github? So that external contributors can see it / work on it; we can always link to this issue from an internal ticket if it becomes a priority

Reply:
Created #169